alws/crud/release.py

```diff
@@ -153,14 +154,25 @@ async def execute_release_plan(release_id: int, db: Session):
 
     async with db.begin():
         release_result = await db.execute(
-            select(models.Release).where(models.Release.id == release_id))
+            select(models.Release
+                   ).where(models.Release.id == release_id).options(
+                selectinload(models.Platform.platform_id)))
         release = release_result.scalars().first()
         if not release.plan.get('packages') or \
                 not release.plan.get('repositories'):
             raise EmptyReleasePlan('Cannot execute plan with empty packages '
                                    'or repositories: {packages}, {repositories}'
                                    .format_map(release.plan))
-
+        for build_id in release.build_ids:
+            try:
+                verified = sign_task.verify_signed_build(
```

Comment:
`verify_signed_build()` is async function, does it work without `await` operator?